functions/register_sso.go

```diff
@@ -117,6 +117,9 @@ func handeServerSSORegisterConn(reqMsg *models.RegisterMsg, apiURI string, conn
 			} else {
 				var response models.RegisterResponse
 				if err := json.Unmarshal(msg, &response); err != nil {
+					fmt.Printf("error: %s\n", err.Error())
```

Comment:
won't this be an unmarshal error, rather than actual error message from server?

Reply:
removed